inference-engine/src/transformations/include/transformations/convert_opset1_to_legacy/convert_mul_or_add_finally.hpp

```diff
@@ -93,9 +96,43 @@ ngraph::graph_rewrite_callback get_callback() {
             }
         }
 
-        // Check that eltwise is not useless otherwise we remove it
-        if ((std::is_same<T, ngraph::opset1::Add>() && ngraph::op::util::constantIsEqualTo(const_node, 0)) ||
-            (std::is_same<T, ngraph::opset1::Multiply>() && ngraph::op::util::constantIsEqualTo(const_node, 1))) {
+        auto constant_broadcast_output = [](ngraph::PartialShape data_pshape, ngraph::Shape const_shape) -> bool {
```

Comment:
Should the `data_pshape` and `const_shape` passed as a constant ref?

Reply:
yes

Reply:
Done